src/test/scala/com/bwsw/cloudstack/entities/dao/AccountDaoTestSuite.scala

```diff
@@ -0,0 +1,129 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.entities.dao
+
+import java.util.UUID
+
+import br.com.autonomiccs.apacheCloudStack.client.ApacheCloudStackRequest
+import br.com.autonomiccs.apacheCloudStack.exceptions.ApacheCloudStackClientRequestRuntimeException
+import com.bwsw.cloudstack.entities.Executor
+import com.bwsw.cloudstack.entities.requests.account.{AccountCreateRequest, AccountFindRequest}
+import com.bwsw.cloudstack.entities.responses.{Account, User}
+import org.scalatest.FlatSpec
+
+class AccountDaoTestSuite extends FlatSpec with TestData {
+  val findRequest = new AccountFindRequest
+
+  "find" should "return an entity list if response json string contains data" in {
```

Comment:
"find" should "return non-empty entity list if a response json string contains the relevant data"

Reply:
ok